pkg/target/regolib/src.rego

```diff
@@ -277,6 +279,24 @@ matches_namespaces(match) {
   count({input.review.namespace} - ns) == 0
 }
 
+doesnt_match_excludednamespaces(match) {
+  not has_field(match, "excludedNamespaces")
+}
+
+doesnt_match_excludednamespaces(match) {
+  has_field(match, "excludedNamespaces")
+  not is_ns(input.review.kind)
+  ns := {n | n = match.excludedNamespaces[_]}
+  count({input.review.namespace} - ns) != 0
+}
+
+doesnt_match_excludednamespaces(match) {
+  has_field(match, "excludedNamespaces")
+  is_ns(input.review.kind)
+  ns := {n | n = match.excludedNamespaces[_]}
+  count({input.review.object.metadata.name} - ns) != 0
```

Comment:
You can use `get_ns[ns]` to get the input object's namespace

Reply:
I guess it would make sense to have additional clauses for `get_ns` to handle the `is_ns` and `not is_ns` cases, if that's what you were meaning? Then we can just have the one `doesnt_match_excludednamespaces` where the field is present.

Reply:
@ritazh I've had a go at moving the logic, could you tell me what you think?